editions/pluginlibrary-server/tiddlers/$__DefaultTiddlers.tid

```diff
@@ -0,0 +1,6 @@
+created: 20131127215321439
+modified: 20210117002817063
+title: $:/DefaultTiddlers
+type: text/vnd.tiddlywiki
+
+[list[$:/StoryList]]
```

Comment:
**System and Config tiddlers will NOT be part of the final PR. ... BUT the make testing this branch simpler!!!**